packages/web/src/common/store/upload/sagas.ts

```diff
@@ -710,7 +714,7 @@ export function* handleUploads({
   if (publishedTrackIds.length === 0) {
     // Errors were reported to sentry earlier in the upload process.
     // Throwing here so callers don't think they succeeded.
-    throw new Error('No tracks were successfully uploaded.')
+    throw new Error('No tracks were successfully uploaded.', { cause: errored })
```

Comment:
This is so nice! 🎉  I don't know why, but I assumed `cause` had to be an `Error`, turns out it can be anything (just like the param to `catch`).

Reply:
Yeah I also learned this! The unfortunate bit about `cause` is that most browsers won't show it in the console.
But since our only goal here is to add additional info that Sentry can use to decorate exceptions, I'm okay with it.